Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -514,6 +480,11 @@ - (void)   toxAV:(OCTToxAV *)toxAV
     [self.audioEngine provideAudioFrames:pcm sampleCount:sampleCount channels:channels sampleRate:sampleRate fromFriend:friendNumber];
 }
 
+- (void)   toxAV:(OCTToxAV *)toxAV bitrateStatusForFriendNumber:(OCTToxFriendNumber)friendNumber
+    audioBitRate:(OCTToxAVAudioBitRate)audioBitrate
+    videoBitRate:(OCTToxAVVideoBitRate)videoBitrate
+{}
```

Comment:
Hm... I think that it should be done **not** in OCTToxAV, but on more higher level. OCTSubmanagerCalls would be perfect place for that. So everything was fine, no need to remove the callback. I was just writing about those empty parentheses.

Can you please get it back (and add a comment to the #88 here)? :-)
Sorry for confusing.